griptape/tools/file_manager/tool.py

```diff
@@ -12,8 +12,8 @@
 
 
 @define
-class FileManager(BaseTool):
-    """FileManager is a tool that can be used to list, load, and save files.
+class FileManagerTool(BaseTool):
+    """FileManagerTool is a tool that can be used to list, load, and save files.
```

Comment:
Not for this PR, but I've always thought that `FileSystemTool` would be more appropriate, since it provides activities that interact with a file system.